SADL/AM-CDM-manufacturingProcess.sadl

```diff
@@ -0,0 +1,56 @@
+uri "http://kdl.ge.com/AM-CDM-manufacturingProcess" alias AM-CDM-manufacturingProcess version "$Revision:$ Last modified on   $Date:$".
+
+import "http://kdl.ge.com/AM-CDM-process".
+import "http://kdl.ge.com/AM-CDM-system".
+import "http://kdl.ge.com/AM-CDM-material".
+
+ManufacturingProcessStep (note "Planned process in which input material is transformed or modified") is a PlannedProcessStep,
+	described by name (note 'Human readable name which designates this process step.') with a single value of type string,
+	described by processStepSpecification with a single value of type ManufacturingSpecification,
+	described by processMachine (note "The machine used to transform the input material.") with a single value of type System,
+	described by controlSoftware (note "Software used to control the process.") with values of type Software,
+	described by consumables (note "Material which is used during the execution of a process, but does not, in a substantial way, become part of material output. Example: Process Gas") with values of type MaterialStock,
+	described by materialInputs (note "Material that is an input to the process and because a part of the material output in a substantial way. Examples: Powder Feedstock, billet, sub-components to be assembled.") with values of type MaterialStock,
+	described by processSimulation (note "Reference to Simulation records associated with this processStep.") with values of type ProcessSimulation.
+
+ManufacturingSpecification (note "A specification for a manufacturing process") is a ProcessStepSpecification,
+	described by globalProcessParameters (note "Nominal system setting which apply to all material stock involved in the Manufacturing Process Step." ) with value of type GlobalProcessParameters,
+	described by partSpecificProcessParameters (note "Nominal system setting which uniquely apply to an individual part to be processed." ) with values of type PartSpecificProcessParameters,
+ 	described by specifiedEnvironment with a single value of type Environment.
+
+// Suggested 'SpecifiedQuality' with upper and lower bounds
+Environment is a class,
+	described by ambientTemperature with a single value of type SpecifiedQuantity,
+	described by ambientPressure with a single value of type SpecifiedQuantity.
+
+GlobalProcessParameters is a class.
+
+PartSpecificProcessParameters is a class,
+	described by orientation (note "Specified orientation for the specific part." ) with a single value of type Orientation,
+	described by design (note "design file which prescribes the changes to the part which are specific to this process." ) with a single value of type Design.
+
+Orientation is a class,
```

Comment:
Orientation and Coordinate still exist in Base; seems redundant

Reply:
Good call! I've replaced the orientation in base.sadl with my updated version. I also noticed that it was replicated in build.sadl and have replaced it there as well